.mk/docker.mk

```diff
@@ -29,7 +26,7 @@ docker-build-local:
 
 .PHONY: docker-push
 docker-push:
-ifdef DOCKER_USERNAME
+ifdef DOCKER_PASSWORD
```

Comment:
```
ifeq (${DOCKER_PASSWORD},)
```

Reply:
Travis unsets DOCKER_PASSWORD

Reply:
yes (if you revert the export statement above)

Reply:
changed